src/stransi/cursor.py

```diff
@@ -58,3 +58,27 @@ class SetCursor(Instruction[CursorMove]):
     """Instruction to set the cursor position."""
 
     move: CursorMove
+
+
+@dataclass
+class CursorVisibilityChange:
```

Comment:
Could you change the name of this class to `CursorVisibility`?

Reply:
Done in 2586d3d